### ISSUE TITLE
Obfuscating a sensitive field

### ISSUE THREAD

**Issue by jnfeinstein** [aq1018/mongoid-history#164]
I would like to track the history of an object with some sensitive fields (like a password).  The value of the change is irrelevant but who changed it and the fact that it changed is important.  Is there a configuration that will handle this?

**Comment by dblock** [comment]
I don't think we have anything built in to handle this, but we should. Labeling as a feature request, feel free to try to implement something.

Obviously, you should never store a password as is :)

**Comment by jnfeinstein** [comment]
I'll start taking a look at this.

I don't see a tag for 0.6.0, is it the current master branch?

**Comment by dblock** [comment]
The 0.6.0 release was made off a fork before we got hold of @aq1018 to move repos. I've tagged it on master, appreciate if you could double-check that I tagged the right revision please.
